x-pack/legacy/plugins/code/public/components/search_page/search.scss

```diff
@@ -1,3 +1,3 @@
-.codeResult__code-block {
+.codeSearch__result {
   margin-bottom: $euiSizeXL;
```

Comment:
@daveyholler is the 12px margin true for all types of search results, or just commits? I was trying to generalize the spacing here.

Reply:
At this point, just commits. The extra space is needed on the other types because there's text that sits outside the panels.

Reply:
Understood, I'll make sure I didn't affect the other results here, and only consolidate if I can.